File: 2540-minimum-common-value/2540-minimum-common-value.js
/**
 * @param {number[]} nums1
 * @param {number[]} nums2
 * @return {number}
 */
var getCommon = function(nums1, nums2) {
    let n1=0
    let n2=0
    while(n1<nums1.length && n2<nums2.length)
    {
        if(nums1[n1]==nums2[n2])
        return nums1[n1]
        else if(nums1[n1]>nums2[n2])n2++
        else n1++
    }
 };

File: 2540-minimum-common-value/2540-minimum-common-value.go
func getCommon(nums1 []int, nums2 []int) int {
    var i,j int=0,0
    for i<len(nums1) && j<len(nums2){
        if nums1[i]>nums2[j]{
            j++} else if nums1[i]< nums2[j]{
                i++
            }else{
                return nums1[i]
            }
    }
    return -1
}